apache/superset
# fix(explore): Collapse padding (cosmetic)

### SUMMARY
It reduces the paddings left and right of the Collapse in the Explore control panel.

### BEFORE
![b4](https://user-images.githubusercontent.com/60598000/117336901-60b14800-aea5-11eb-9544-f2a5a69954df.png)

### AFTER
<img width="803" alt="Screen Shot 2021-05-10 at 18 59 40" src="https://user-images.githubusercontent.com/60598000/117688908-e98bf480-b1c1-11eb-9218-4db956f19ef6.png">

### TEST PLAN
1. Open a chart in Explore
2. Observe the padding-left and right of the control panel

### ADDITIONAL INFORMATION


- [x] Has associated issue: #12634
